File: Week 1/Cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>
#define quarter 25
#define dime 10
#define nickel 5
#define penny 1
int main(void)
{
float cash;
do{
cash= get_float("Change owed: ");
}while(cash <= 0);
int money= round (cash*100);
int counter=0;
while(money != 0)
{
if(money >= quarter)
{
money= money-quarter;
counter++;
}
else if(money >= dime)
{
money= money-dime;
counter++;
}
else if(money >= nickel)
{
money= money-nickel;
counter++;
}
else
{
money = money-penny;
counter++;
}
}
printf("%d\n", counter);
return 0;
}

File: Week 6/cash.py
from cs50 import get_float

money = 0.0
while(True):
    money = get_float("Change owed: ")
    if (money > 0.0):
        break

ans = 0
peni = money * 100
while(peni != 0):

    if peni >= 25:
        peni -= 25
        ans += 1
    elif peni >= 10:
        peni -= 10
        ans += 1
    elif peni >= 5:
        peni -= 5
        ans += 1
    else:
        peni -= 1
        ans += 1


print(ans)
